x-pack/plugins/ml/public/explorer/explorer_charts/explorer_chart_info_tooltip.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import './styles/explorer_chart_info_tooltip.less';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { CHART_TYPE } from '../explorer_constants';
+
+const CHART_DESCRIPTION = {
+  [CHART_TYPE.EVENT_DISTRIBUTION]: 'The gray dots depict the distribution of occurences over time for a sample of by_field_values with \
+more frequent event types at the top and more rare ones at the bottom.',
+  [CHART_TYPE.POPULATION_DISTRIBUTION]: 'The gray dots depict the distribution of values over time for a sample of over_field_values.'
+};
+
+import { EuiSpacer } from '@elastic/eui';
+
+function TooltipDefinitionList({ toolTipData }) {
+  return (
+    <dl className="mlDescriptionList">
```

Comment:
Could we go with `EuiDescriptionList` here, or are there problems when it is used inside a tooltip?

Reply:
For now I went with custom styles inspired by `EuiDescriptionList`, I had issues getting it work as intended within a tooltip. I added a note to #21163 about it.